viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1569,10 +1570,12 @@ export class Cognite3DViewer {
       const camera = this.cameraManager.getCamera();
       TWEEN.update(time);
       this.recalculateBoundingBox();
-      this._activeCameraManager.update(
-        this.cameraManagerClock.getDelta(),
-        this._updateNearAndFarPlaneBuffers.combinedBbox
-      );
+
+      if (this._activeCameraManager instanceof FlexibleCameraManager) {
```

Comment:
I don't think this would work - `this._activeCameraManager` is of type `ProxyCameraManager`. We need to check `this._activeCameraManager.innerCameraManager`.

Reply:
Fixed